pkg/virtual/framework/dynamic/types.go

```diff
@@ -51,3 +53,10 @@ func (vw *DynamicVirtualWorkspace) IsReady() error {
 func (vw *DynamicVirtualWorkspace) ResolveRootPath(urlPath string, context context.Context) (accepted bool, prefixToStrip string, completedContext context.Context) {
 	return vw.RootPathResolver(urlPath, context)
 }
+
+func (vw *DynamicVirtualWorkspace) Authorize(ctx context.Context, a authorizer.Attributes) (authorized authorizer.Decision, reason string, err error) {
+	if vw.Authorizer != nil {
+		return vw.Authorizer(ctx, a)
+	}
+	return authorizer.DecisionAllow, "", nil
```

Comment:
Should this be NoOpinion?

Reply:
So to avoid breaking anything we'd add an explicit AllowEverything authorizers in all the virtual workspace instances ?

Reply:
Also a vw without authorizer should forbid every request.

Reply:
Changed in https://github.com/kcp-dev/kcp/pull/1186/commits/cf98b013c8b5f369eb3cfa950de041360f7f0e88